java/spatial-operations/README.md

```diff
@@ -1,22 +1,41 @@
-# Spatial Operations
-Perform geometry-on-geometry spatial operations by using `GeometryEngine`. Two input `Polygons` are added as `Graphics` to a `GraphicsOverlay` and displayed in a `MapView`. 
+# Perform spatial operations
 
-![Spatial Operations App](spatial-operations.png)
+Find the union, intersection, or difference of two geometries.
+
+![Image of perform spatial operations](spatial-operations.png)
+
+## Use case
+
+The different spatial operations (union, difference, symmetric difference, and intersection) can be used for a variety of spatial analyses. For example, government authorities may use the intersect operation to determine whether a proposed road cuts through a restricted piece of land such as a nature reserve or a private property.
+When these operations are chained together, they become even more powerful. An analysis of food deserts within an urban area might begin by union-ing service areas of grocery stores, farmer's markets, and food co-ops. Taking the difference between this single geometry of all services areas and that of a polygon delineating a neighborhood would reveal the areas within that neighborhood where access to healthy, whole foods may not exist.
 
 ## How to use the sample
-Using the options menu, different spatial operations can be performed, the result of which is shown as a third `Graphic`, in red, in the `MapView`. Additionally, the `union` method is used to create an initial geometry with which to set the `Viewpoint` of the `MapView`.
+
+The sample provides an option to select a spatial operation. When an operation is selected, the resulting geometry is shown in red. The 'reset operation' button undoes the action and allow selecting a different operation.
 
 ## How it works
-The `GeometryEngine` methods `difference`, `intersection`, `symmetricDifference`, and `union` all perform geometry-on-geometry operations against two `Geometry` objects. The result is represented by a third `Geometry` that is returned from the method call.
+
+1.  Create a `GraphicsOverlay` and add it to the `MapView`.
```

Comment:
This implementation actually creates two graphics overlays; one for the input and one for the result. However, I think we are supposed to leave this fairly high level, so it's probably okay to leave it as is. Up to you and @Rachael-E .

Reply:
I took a swing at addressing this. Changes are in steps 1 and 5.